crates/bevy_winit/src/lib.rs

```diff
@@ -142,7 +142,7 @@ impl<T: Event> Plugin for WinitPlugin<T> {
 #[derive(Debug, Default, Clone, Copy, Event)]
 pub struct WakeUp;
 
-/// The [`winit::event_loop::EventLoopProxy`] with the specific [`winit::event::Event::UserEvent`] used in the [`winit_runner`].
+/// A re-export of [`winit::event_loop::EventLoopProxy`].
```

Comment:
The old description didn't seem to make much sense, and also it referenced `winit_runner` which is private.